README.md

```diff
@@ -1,12 +1,21 @@
-#This project was abandoned because [Moxie Marlinspike said he was not OK with LibreSignal using the Open Whisper Systems servers and the name "Signal"](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165).
+#This GitHub repository has been abandoned.
```

Comment:
Just for the record I have been distributing independent Signal builds named LibreSignal (in my main repository on https://fdroid.eutopia.cz) long before LibreSignal (WebSocket fork) project has been created.